MANUAL_REGRESSION.md

```diff
@@ -242,15 +262,15 @@ Outcome:
 5. Click `Enable webcam`
 6. You should see the capture screen and camera permissions prompt
 
-##### 19. Check permission priming screen does not display when webcam is available and permission was already granted
+##### 20. Check permission priming screen does not display when webcam is available and permission was already granted
 (on Firefox, Safari and Microsoft Edge browsers)
```

Comment:
There browsers display the screen multiple times, unless the user has changed the browser settings to remember the choice. Applause keeps flagging this as a bug, so I think we should change it